CHANGELOG.md

```diff
@@ -37,6 +37,10 @@
 
 - [#5867](https://github.com/ChainSafe/forest/pull/5867) Added `--unordered` to `forest-cli snapshot export` for exporting `CAR` blocks in non-deterministic order for better performance with more parallelization.
 
+- [#5946](https://github.com/ChainSafe/forest/pull/5946) Added `--n-epochs` to `forest-cli state compute` for computating state trees in batch.
```

Comment:
There is also a breaking change to the `Forest.StateCompute` API, no?

Reply:
> There is also a breaking change to the Forest.StateCompute API, no?

It is but as I understand `Forest.*` are internal RPC methods for CLI tools that we don't garantee backward compatibility.